src/components/sequence.jsx

```diff
@@ -81,65 +100,85 @@ class Sequence extends Component {
       </DropdownButton>
     );
   };
+  const chunks = getChunks(sequence, chunkSize, textSize);
 
-  render() {
-    const { sequence, chunkSize } = this.props;
-    const { textSize, copied } = this.state;
-    const chunks = this.getChunks(sequence, chunkSize, textSize);
-    let content;
-    // If textSize was not provided, add a text element so it can be measured
-    // in componentDidMount
-    if (textSize === null) {
-      content = (
-        <svg>
-          <text
-            ref={(t) => {
-              this.text = t;
-            }}
-          >
-            M
-          </text>
-        </svg>
-      );
-    } else {
-      content = (
-        <Fragment>
-          {chunks.map(chunk => (
-            <span className="sequence__sequence__chunk" key={`chunk_${v1()}`}>
-              {chunk}
-            </span>
-          ))}
-        </Fragment>
-      );
-    }
-
-    return (
-      <Fragment>
-        <div className="action-bar">
-          {this.getSelectors()}
-          <CopyToClipboard text={sequence} onCopy={() => this.setState({ copied: true })}>
-            <button type="button" className="button sequence__copy-button">
-              {copied ? 'Copied' : 'Copy sequence'}
-            </button>
-          </CopyToClipboard>
-        </div>
-        <div className="sequence">
-          <div className="sequence__sequence">{content}</div>
+  return (
+    <Fragment>
+      <div className="action-bar button-group">
+        <DropdownButton label="Tools">
+          <ul className="no-bullet">
+            <li>
+              <a href={`/blast/accession/${accession}`}>BLAST</a>
```

Comment:
Do want to give the consumers the option to pass in the path to their blast endpoint?

Reply:
Good idea! Added